app/models/concerns/audit_associations.rb

```diff
@@ -1,15 +1,11 @@
 module AuditAssociations
-  def self.included(base)
-    base.send :include, InstanceMethods
-  end
-
-  module InstanceMethods
+  module AssociationsChanges
     def changes_to_save
-      if audited_options[:associations].present?
-        super.merge(associated_changes)
```

Comment:
Since we now normalize the option, we can always merge - if there isn't any associations, it will just merge with an empty hash.